frontend/src/metabase/admin/databases/components/DatabaseEditApp/Sidebar/Sidebar.tsx

```diff
@@ -59,6 +75,15 @@ const DatabaseEditAppSidebar = ({
     [database, dismissSyncSpinner],
   );
 
+  const handleToggleModelActionsEnabled = useCallback(
+    (nextValue: boolean) =>
+      updateDatabase({
+        id: database.id,
+        settings: { "database-enable-actions": nextValue },
+      }),
+    [database, updateDatabase],
```

Comment:
not sure why it has to depend on the whole `database` when this callback only depends on `database.id`. I could have the same questions for the other callbacks too.

Reply:
That's Copilot, not me! 😄